app/init.php

```diff
@@ -228,6 +229,11 @@
 
 App::setMode(App::getEnv('_APP_ENV', App::MODE_TYPE_PRODUCTION));
 
+$isProduction = App::getEnv('_APP_ENV', App::MODE_TYPE_DEVELOPMENT) === App::MODE_TYPE_PRODUCTION;
```

Comment:
You can use this method instead: https://github.com/utopia-php/framework/blob/master/src/App.php#L395, no need for variable or function in most cases when that var/functions are used just once.